modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -74,10 +73,10 @@
 	alt_titles = list("Detective", "Forensic Technician", "Private Investigator", "Forensic Scientist")
 
 /datum/job/doctor
-	alt_titles = list("Medical Doctor", "Surgeon", "Nurse")
+	alt_titles = list("Medical Doctor", "Surgeon", "Nurse", "General Practitioner", "Medical Resident", "Physician", "Podiatrist")
```

Comment:
While it's amusing, I'm not sure "podiatrist" will actually bring anything beneficial.

Reply:
forgot that
upsetting, i wanted the hubert joke